lib/index.js

```diff
@@ -61,38 +61,41 @@ function GitHub (options) {
  */
 GitHub.prototype.req = function (url, data, callback) {
     var self = this
-      , req = null
-      , url = self.host + url
-      ;
+        , req = null
+        , url = self.host + url
+        ;
 
     if (typeof data === "function") {
         callback = data;
         data = undefined;
     }
 
     // Handle the token
-    if (self.token) {
-        url += url.indexOf("?") > -1 ? "&" : "?";
-        url += "access_token=" + self.token;
-    }
+    // if (self.token) {
+    //     url += url.indexOf("?") > -1 ? "&" : "?";
+    //     url += "access_token=" + self.token;
+    // }
 
     // Jsonreq on server, XHR on client
     if (typeof Jsonreq === "function") {
         req = Jsonreq({
             url: url
-          , data: data
-          , headers: {
+            , Authentication: `Bearer ${self.token}`
```

Comment:
Can you please make this optional?

Reply:
I don't know how to, i'm some what new to contributing. But i don't see why, as github will deprecate authentication via query parameters soon anyways.

Reply:
Hey, you still there?